frontend/dispatcher.lua

```diff
@@ -1,3 +1,30 @@
+--[[--
+Dispatcher module
+
+To add a new action an entry must be added to `settingsList` & `dispatcher_menu_order`
+This can also be done at runtime via `registerAction`
```

Comment:
You can make that a direct reference btw, something like `{@registerAction}` otoh, or with a pipe if you want to customize the text.

Reply:
right now it is a direct link.
 {@registerAction} doesn't do anything

Reply:
Um, it most certainly does. :-P

Reply:
I am using ldoc 1.4.6 from archlinux with lua 5.4

Reply:
your comment here has a typo `{@}` further down you wrote the correct way `@{}` It is working now. :)